examples/blob/main.py

```diff
@@ -0,0 +1,22 @@
+from pyteal import *
+from pytealutils.storage.blob import Blob
+
+
+def test():
+    b = Blob()
+
+    test = Seq(
+        b.zero(),  # Required on initialization
+        Pop(b.write(Int(0), Int(0), Bytes("deadbeef" * 16))),
+        Log(b.read(Int(0), Int(8), Int(32))),
```

Comment:
Maybe worth storing `b.read(Int(0), Int(8), Int(32))` and then asserting its value to be what you expect it to be, in addition to logging